### PR TITLE
chore(docs): p-files Part2 - adjust headlines according to style guide + adjust some texts

### PR DESCRIPTION
**Pull request by codingCookie**
## Description
This PR introduces some changes to **first part** of the **p-files** in the **docs** folder.

#### What have i done?
- changed some headlines to fit the style guide
- a few text changes

## Related Issues
Related to https://github.com/gatsbyjs/gatsby/issues/18284

### CODE REVIEW

File: docs/docs/preoptimizing-images.md [hunk 1]
```diff
@@ -2,14 +2,18 @@
...
+Gatsby ships with excellent image optimization capabilities (see the [image tutorial](/tutorial/image-tutorial/) for more info). However, this image optimization can come with a cost. It can be fairly CPU intensive, and in some cases may lead to long build times. As a means of debugging and perhaps improving your overall build performance, it _may_ be helpful to pre-optimize your (extremely large) images.
 
-First, some context. `gatsby-plugin-sharp` ships with a `fluid` option which will attempt to create four images intended to map to various screen resolutions. Producing multiple images ensures that your images are ready and optimized for phone displays, desktop displays, and everything in between. This plugin receives a non-optimized image and produces optimized images for _all_ of your users and devices. Magic!
+## First, some context.
```
Comment (LekoArts) [inline docs/docs/preoptimizing-images.md]: Please revert this change as this document doesn't profit from too many headings and I feel like this heading doesn't improve the reading (as it's also non-descriptive).

Comment (codingCookie) [reply docs/docs/preoptimizing-images.md]: sure i will revert it - no problem :)
